examples/powder_averaging.jl

```diff
@@ -7,7 +7,6 @@
 
 using Sunny, GLMakie
 using Statistics: mean
-#md Makie.inline!(true); #hide
```

Comment:
Was this a no-op?

Reply:
It was previously needed as a workaround for this bug https://github.com/MakieOrg/Makie.jl/issues/2805.